portal/views/tou.py

```diff
@@ -66,7 +89,63 @@ def get_tou(user_id):
     if not user:
         abort(404)
     current_user().check_role(permission='view', other_id=user_id)
-    tous = ToU.query.join(Audit).filter(Audit.user_id==user_id).first()
+
+    tous = ToU.query.join(Audit).filter(Audit.user_id == user_id)
+
+    return jsonify(tous=[d.as_json() for d in tous])
+
+
+@tou_api.route('/user/<int:user_id>/tou/<string:tou_type>')
+@oauth.require_oauth()
+def get_tou_by_type(user_id, tou_type):
+    """Access Terms Of Use info for given user & ToU type
+
+    Returns ToU{'accepted': true|false} for requested user given the
```

Comment:
I think we should make these two APIs more consistent - lets return the same JSON for the one, if accepted.  Not sure how to best handle the false case - ideas?